Phonebook.Frontend/src/app/pages/settings/settings.component.html

```diff
@@ -28,8 +36,9 @@ <h1 i18n="SettingsComponent|Title of the settings page@@SettingsComponentTitle">
           </mat-form-field>
         </mat-list-item>
       </mat-list>
-      <h2 i18n="SettingsComponent|Color Theme SubTitle@@SettingsComponentSubTitleColorTheme"
-        >Color Theme</h2
+      <mat-card-subtitle
```

Comment:
This should only be used once in a mat-card. Sry, if I might have suggested using that before. Can you use <h3> or one of the typography classes? https://material.angular.io/guide/typography

Reply:
In the latest commit I used h4. It looks as though the style and size is pretty much the same to the mat-card-subtitle. However, the writing is now black and there is no typography class to change it to grey either. So, we could leave it black or add additional CSS to make it look as similar to the mat-card-subtitle as possible.

Reply:
black is ok